Tools/cases_generator/lexer.py

```diff
@@ -240,7 +240,12 @@ def to_text(tkns: list[Token], dedent: int = 0) -> str:
             res.append('\n')
             col = 1+dedent
         res.append(' '*(c-col))
-        res.append(tkn.text)
+        text = tkn.text
+        if dedent != 0 and tkn.kind == 'COMMENT' and '\n' in text:
+            if dedent < 0:
+                text = text.replace('\n', '\n' + ' '*-dedent)
+            # TODO: dedent > 0
```

Comment:
Yeah, I inherited that from the lexer. I'll eventually fix it. For a while it was inconvenient because there was also a variable named `indent` in a lot of places. That's now `self.indent`. (But still, it's a string of spaces rather than an int.)